b/babel-plugin-transform-runtime/babel-plugin-transform-runtime_ubi8_3.sh

```diff
@@ -0,0 +1,44 @@
+# ----------------------------------------------------------------------------
+#
+# Package       : babel-plugin-transform-runtime
+# Version       : v7.4.0
+# Source repo   : https://github.com/babel/babel
+# Tested on     : UBI 8.3
+# Script License: Apache-2.0 License    
+# Maintainer    : Varsha Aaynure <Varsha.Aaynure@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+#Variables
+PACKAGE_URL=https://github.com/babel/babel.git
+PACKAGE_VERSION="${1:-v7.4.0}"
+
+yum update -y 
+
+#Install required files
+yum install -y git 
+
+#To install node 12 
+wget "https://nodejs.org/dist/v12.22.4/node-v12.22.4-linux-ppc64le.tar.gz"
```

Comment:
`wget` needs to be installed before its usage.
please add `wget` to `yum install` list

or

If you can do it with `curl`, it is always appreciated.

Reply:
okay will add wget to yum install list.